setup.cfg

```diff
@@ -0,0 +1,18 @@
+[flake8]
+doctests = True
```

Comment:
this is cute! if I get it right - flake8 would run doctests in the code? then could you please also add .github workflow to run flake8?  folks

Reply:
No, that means that `flake8` will apply its linting to doctests.

Reply:
ah, so it would still need some `pytest --doctest-modules`? please add a run even though there is no other dedicated tests besides those doctests

Reply:
Done.